### ISSUE TITLE
Bird Species Classification using Deep Learning

### ISSUE THREAD

**Issue by pavitraag** [abhisheks008/DL-Simplified#843]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **BIRD SPECIES CLASSIFICATION with DEEP LEARNING** : </br>
:red_circle: **Aim** : Classify the bird species </br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/gpiosenka/100-bird-species/data?select=birds.csv </br>
:red_circle: **Approach** : EfficientNetB0, InceptionV3, VGG16 model, InceptionResNetV2 2

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Pavitraa G
- GitHub Profile Link : @pavitraag
- Email ID : pavitraagopi@gmail.com
- What is your participant role? GSSOC 24 Contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Assigned @pavitraag

**Comment by ojaswichopra** [comment]
can you please assign this issue to me?
I really want to work on it

**Comment by abhisheks008** [comment]
> can you please assign this issue to me? I really want to work on it

Already assigned to another contributor.

**Comment by IkkiOcean** [comment]
Can you assign this to me? 
Full name : Vivek Prakash
GitHub Profile Link : https://github.com/IkkiOcean
Email ID : vivekprakash.st@gmail.com
participant role : GSSOC 24 Contributor

**Comment by abhisheks008** [comment]
Hi @IkkiOcean can you please share your approach for solving this issue?

**Comment by IkkiOcean** [comment]
I have chosen the InceptionResNetV2 approach since bird species classification is a fine-grained task that often requires distinguishing subtle differences in features such as feather patterns, beak shapes, or colors. The Inception modules make it particularly effective at capturing these intricate details.

**Comment by abhisheks008** [comment]
> I have chosen the InceptionResNetV2 approach since bird species classification is a fine-grained task that often requires distinguishing subtle differences in features such as feather patterns, beak shapes, or colors. The Inception modules make it particularly effective at capturing these intricate details. @abhisheks008

As you can see all the projects present here consist of at least 3-4 deep learning methods implemented. Based on the accuracy scores of the each model, we can conclude the best fitted model for this dataset.

Similarly can you please update your approach and let me know.

**Comment by IkkiOcean** [comment]
> > I have chosen the InceptionResNetV2 approach since bird species classification is a fine-grained task that often requires distinguishing subtle differences in features such as feather patterns, beak shapes, or colors. The Inception modules make it particularly effective at capturing these intricate details. @abhisheks008
> 
> As you can see all the projects present here consist of at least 3-4 deep learning methods implemented. Based on the accuracy scores of the each model, we can conclude the best fitted model for this dataset.
> 
> Similarly can you please update your approach and let me know.

Sure thing! I’ll begin working on it and will update you on my approaches shortly.

**Comment by ramana2074** [comment]
Can you assign this to me?
Full name : Venkata Ramana
GitHub Profile Link : https://github.com/ramana2074
Email ID : billanavenkataramana@gmail.com
participant role : GSSOC 24 Contributor

Approach : 
process the images (resize, normalize, convert sounds to spectrograms), and use convolutional neural networks (CNNs) for image classification or spectrogram analysis. Train the model, evaluate performance, and fine-tune for bird species identification.

**Comment by IkkiOcean** [comment]
> > I have chosen the InceptionResNetV2 approach since bird species classification is a fine-grained task that often requires distinguishing subtle differences in features such as feather patterns, beak shapes, or colors. The Inception modules make it particularly effective at capturing these intricate details. @abhisheks008
> 
> As you can see all the projects present here consist of at least 3-4 deep learning methods implemented. Based on the accuracy scores of the each model, we can conclude the best fitted model for this dataset.
> 
> Similarly can you please update your approach and let me know.

After analyzing the data and its properties, I believe I will be able to explore all the approaches mentioned, including EfficientNetB0, InceptionV3, VGG16, and InceptionResNetV2. So, please assign this issue to me under GSSOC-extd with the appropriate level and tag it for Hacktoberfest.

**Comment by shivam060404** [comment]
i would to like to work on this project.kindly assigned me please

**Comment by abhisheks008** [comment]
> > > I have chosen the InceptionResNetV2 approach since bird species classification is a fine-grained task that often requires distinguishing subtle differences in features such as feather patterns, beak shapes, or colors. The Inception modules make it particularly effective at capturing these intricate details. @abhisheks008
> > 
> > 
> > As you can see all the projects present here consist of at least 3-4 deep learning methods implemented. Based on the accuracy scores of the each model, we can conclude the best fitted model for this dataset.
> > Similarly can you please update your approach and let me know.
> 
> After analyzing the data and its properties, I believe I will be able to explore all the approaches mentioned, including EfficientNetB0, InceptionV3, VGG16, and InceptionResNetV2. So, please assign this issue to me under GSSOC-extd with the appropriate level and tag it for Hacktoberfest.

Thanks for sharing your approach. You can start working on it. This issue is assigned to you @IkkiOcean

**Comment by IkkiOcean** [comment]
> > > > I have chosen the InceptionResNetV2 approach since bird species classification is a fine-grained task that often requires distinguishing subtle differences in features such as feather patterns, beak shapes, or colors. The Inception modules make it particularly effective at capturing these intricate details. @abhisheks008
> > > 
> > > 
> > > As you can see all the projects present here consist of at least 3-4 deep learning methods implemented. Based on the accuracy scores of the each model, we can conclude the best fitted model for this dataset.
> > > Similarly can you please update your approach and let me know.
> > 
> > 
> > After analyzing the data and its properties, I believe I will be able to explore all the approaches mentioned, including EfficientNetB0, InceptionV3, VGG16, and InceptionResNetV2. So, please assign this issue to me under GSSOC-extd with the appropriate level and tag it for Hacktoberfest.
> 
> Thanks for sharing your approach. You can start working on it. This issue is assigned to you @IkkiOcean

sure. I will be making the PR soon as I am already done with the project, just need some tinkering with readme

**Comment by IkkiOcean** [comment]
@abhisheks008 it's been 4 days sincemy pull requt. Can you please look into it
